ui/package.json

```diff
@@ -13,7 +13,9 @@
         "node": "20.x"
     },
     "scripts": {
-        "dev": "concurrently \"NEXT_PUBLIC_VERCEL_GIT_COMMIT_REF=local next dev -p 3000\" \"local-ssl-proxy --source 3001 --target 3000\"",
+        "dev": "concurrently npm:dev:*",
```

Comment:
Copied from keystone gateway, thought I may as well include it in this work.

Reply:
So you have concurrently run two npm scripts, one of which is also running concurrently? Maybe dev:next could be split out? Though I don't know how "in-support" the local-ssl-proxy library is given it's last update was a year ago

Reply:
Simplified this with the nextJS `--experimental-https` flag which also got rid of the local-ssl-proxy package.